packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/connectors.scss

```diff
@@ -0,0 +1,27 @@
+ovh-manager-pci-project-database-connectors {
```

Comment:
Same css is repeated in packages/manager/modules/pci/src/projects/project/storages/databases/components/restart-button/restart-button.scss
Is there a way to reuse it?

Reply:
At first, restart-button css was in this file, I forgot to remove it when I moved the restart button to its own component. I cleaned it up here: https://github.com/ovh/manager/pull/6577/commits/04eb3070245b89af0b56b5f0a7d8b484481395de